src/components/Search.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import Button from './Button';
+
+export default class Search extends React.PureComponent {
+    constructor(props) {
```

Comment:
If you're not setting the state in the constructor, or not doing some other logic, there is no need to write a constructor function. 'super' will be called automatically

Reply:
Done